TypeError
Python 3.9.7: E:\Anaconda3\envs\imgproapp\python.exe
Thu Dec  9 12:04:25 2021

A problem occurred in a Python script.  Here is the sequence of
function calls leading up to the error, in the order they occurred.

 E:\BaiduNetdiskWorkspace\Cloud_PythonProgramming\img-proc-app\gui\maingui_edit.py in about(self=<gui.maingui_edit.Ui_MainWindow object>)
  619         关于槽函数
  620         '''
  621         self.AboutDialog = Ui_About()
  622         self.AboutDialog.setFixedSize(self.AboutDialog.width(), self.AboutDialog.height())
  623         self.AboutDialog.exec_()
self = <gui.maingui_edit.Ui_MainWindow object>
self.AboutDialog undefined
global Ui_About = <class 'gui.about_edit.Ui_About'>

 E:\BaiduNetdiskWorkspace\Cloud_PythonProgramming\img-proc-app\gui\about_edit.py in __init__(self=<gui.about_edit.Ui_About object>)
    7     def __init__(self):
    8         super(Ui_About, self).__init__()
    9         self.setupUi(self)  # 括号里必须有self
   10 
   11     def setupUi(self, About_Dialog):
self = <gui.about_edit.Ui_About object>
self.setupUi = <bound method Ui_About.setupUi of <gui.about_edit.Ui_About object>>

 E:\BaiduNetdiskWorkspace\Cloud_PythonProgramming\img-proc-app\gui\about_edit.py in setupUi(self=<gui.about_edit.Ui_About object>, About_Dialog=<gui.about_edit.Ui_About object>)
   22         self.label_Info.setGeometry(QtCore.QRect(10, 110, 391, 181))
   23         font = QtGui.QFont()
   24         font.setStyle('QLabel { font-size:12px; }')
   25         self.label_Info.setFont(font)
   26         self.label_Info.setAlignment(QtCore.Qt.AlignLeading | QtCore.Qt.AlignLeft | QtCore.Qt.AlignVCenter)
font = <PyQt5.QtGui.QFont object>
font.setStyle = <built-in method setStyle of QFont object>
TypeError: setStyle(self, QFont.Style): argument 1 has unexpected type 'str'
    __cause__ = None
    __class__ = <class 'TypeError'>
    __context__ = None
    __delattr__ = <method-wrapper '__delattr__' of TypeError object>
    __dict__ = {}
    __dir__ = <built-in method __dir__ of TypeError object>
    __doc__ = 'Inappropriate argument type.'
    __eq__ = <method-wrapper '__eq__' of TypeError object>
    __format__ = <built-in method __format__ of TypeError object>
    __ge__ = <method-wrapper '__ge__' of TypeError object>
    __getattribute__ = <method-wrapper '__getattribute__' of TypeError object>
    __gt__ = <method-wrapper '__gt__' of TypeError object>
    __hash__ = <method-wrapper '__hash__' of TypeError object>
    __init__ = <method-wrapper '__init__' of TypeError object>
    __init_subclass__ = <built-in method __init_subclass__ of type object>
    __le__ = <method-wrapper '__le__' of TypeError object>
    __lt__ = <method-wrapper '__lt__' of TypeError object>
    __ne__ = <method-wrapper '__ne__' of TypeError object>
    __new__ = <built-in method __new__ of type object>
    __reduce__ = <built-in method __reduce__ of TypeError object>
    __reduce_ex__ = <built-in method __reduce_ex__ of TypeError object>
    __repr__ = <method-wrapper '__repr__' of TypeError object>
    __setattr__ = <method-wrapper '__setattr__' of TypeError object>
    __setstate__ = <built-in method __setstate__ of TypeError object>
    __sizeof__ = <built-in method __sizeof__ of TypeError object>
    __str__ = <method-wrapper '__str__' of TypeError object>
    __subclasshook__ = <built-in method __subclasshook__ of type object>
    __suppress_context__ = False
    __traceback__ = <traceback object>
    args = ("setStyle(self, QFont.Style): argument 1 has unexpected type 'str'",)
    with_traceback = <built-in method with_traceback of TypeError object>

The above is a description of an error in a Python program.  Here is
the original traceback:

Traceback (most recent call last):
  File "E:\BaiduNetdiskWorkspace\Cloud_PythonProgramming\img-proc-app\gui\maingui_edit.py", line 621, in about
    self.AboutDialog = Ui_About()
  File "E:\BaiduNetdiskWorkspace\Cloud_PythonProgramming\img-proc-app\gui\about_edit.py", line 9, in __init__
    self.setupUi(self)  # 括号里必须有self
  File "E:\BaiduNetdiskWorkspace\Cloud_PythonProgramming\img-proc-app\gui\about_edit.py", line 24, in setupUi
    font.setStyle('QLabel { font-size:12px; }')
TypeError: setStyle(self, QFont.Style): argument 1 has unexpected type 'str'

